apps/forklift/docker-compose.yml

```diff
@@ -88,8 +88,8 @@ services:
     healthcheck:
       test: ["CMD-SHELL", "kafka-topics.sh --bootstrap-server 127.0.0.1:9092 --list"]
       interval: 10s
-      timeout: 20s
-      retries: 3
+      timeout: 30s
```

Comment:
Match trino's healthcheck time so that integration tests don't fail on M1 chips, which are slower to start docker images apparently